exercises/K-functions-parameters/exercise3.js

```diff
@@ -1,5 +1,17 @@
 // Write your function here
+// option 1
+function createGreeting(fname, salam) {
+    return fname + salam;
+}
+var greeting = createGreeting(' Hi ', 'my name is Adam');
+console.log(greeting);
 
-var greeting = createGreeting("Daniel");
+// option 2
+function createGreeting2(fname, salam) {
+    fname = "Adam";
+    salam = " Good morning "
```

Comment:
salam isn't very clear in meaning Might call it something like salutation or greeting